weblab_cg/tests/test_chaste_cg.py

```diff
@@ -12,6 +12,13 @@
 LOGGER.setLevel(logging.DEBUG)
 
 
+def pytest_configure(config):
+    """ Sets up pytest configuration programatically and adds pytest marks so we can use the, below."""
+    config.addinivalue_line(
+        'markers', 'env(chaste): tests specific for chaste code gen, exclude these with pytest -m "not chaste"'
```

Comment:
Did you check that using `-m` works as expected?

Reply:
of course!